tweetflask.py

```diff
@@ -97,18 +97,18 @@ def challenge(self):
 
     basic_auth = ChallengeAuth(app)
     with open('config.yaml', 'r') as yfile:
-        yamldata = yaml.load(yfile)
-    secureProtocol = yamldata['server']['secureProtocol']
+        yaml_data = yaml.load(yfile)
+    secureProtocol = yaml_data['server']['secureProtocol']
     try:
         context = ssl.SSLContext(getattr(ssl, secureProtocol))
     except AttributeError as e:
```

Comment:
flake8 warning: `local variable 'e' is assigned to but never used. [F841]`

Reply:
if i don't use "as e" in the except then it will throw me an error

Reply:
What error?